tabbycat/notifications/utils.py

```diff
@@ -126,16 +128,18 @@ def standings_email_generator(url, round_id):
     context['URL'] = url if tournament.pref('public_team_standings') else ""
 
     for team in teams:
-        context['POINTS'] = str(team.points_count)
-        context['TEAM'] = team.short_name
+        context_team = context.copy()
+        context_team['POINTS'] = str(team.points_count)
+        context_team['TEAM'] = team.short_name
 
         for speaker in team.speaker_set.all():
             if speaker.email is None:
                 continue
 
-            context['USER'] = speaker.name
+            context_user = context.copy()
```

Comment:
did you intend to copy `context_team` here?

Reply:
Good catch.